src/hotspot/share/interpreter/oopMapCache.cpp

```diff
@@ -399,38 +402,28 @@ void OopMapCacheEntry::deallocate(OopMapCacheEntry* const entry) {
 
 // Implementation of OopMapCache
 
-void InterpreterOopMap::resource_copy(OopMapCacheEntry* from) {
-  assert(_resource_allocate_bit_mask,
-    "Should not resource allocate the _bit_mask");
-  assert(from->has_valid_mask(),
-    "Cannot copy entry with an invalid mask");
+void InterpreterOopMap::copy_from(OopMapCacheEntry* src) {
```

Comment:
Possibly for another  RFE: src pointer should be const

Reply:
Fixed, should be fine to do it in this PR.